README.md

```diff
@@ -1,3 +1,9 @@
+# Archived
+
+After months developing Fleek Network in this repository, we have realized the need to reboot the codebase, our engineers are actively working harder than ever on the upcoming new iteration of the protocol (it's gonna be cool 😎 ). We appreciate all of the love you have given to this repository with your stars. But since we have no activity here, we have decided to archive this repository, and let it live forever as a part of the history of Fleek Network.
```

Comment:
```suggestion
After months developing Fleek Network in this repository, we have realized the need to reboot the codebase, our engineers are actively working harder than ever on the upcoming new iteration of the protocol (it's gonna be cool 😎). We appreciate all of the love you have given to this repository with your stars. But since we have no activity here, we have decided to archive this repository, and let it live forever as a part of the history of Fleek Network.
```